src/main/java/reposense/builder/ConfigurationBuilder.java

```diff
@@ -9,8 +9,8 @@
 public class ConfigurationBuilder {
     private RepoConfiguration config;
 
-    public ConfigurationBuilder(String organization, String repoName, String branch) {
-        config = new RepoConfiguration(organization, repoName, branch);
+    public ConfigurationBuilder(String location, String branch) {
```

Comment:
`location` sounds abit ambigious (what location?)
maybe can be more specific `repoLocation`?

Reply:
Ah.. the `location` is from a csv config file.
Should I keep it faithful to the column header of the csv config file?